### PR TITLE
Feature/failed notifications

### PR DESCRIPTION
**Pull request by stevenbal**
I used django-db-logger as Sergei suggested https://github.com/open-zaak/open-zaak/pull/327#pullrequestreview-333327042

issue https://github.com/open-zaak/open-zaak/issues/310
requires https://github.com/VNG-Realisatie/vng-api-common/pull/120 to be merged and bumped first

### CODE REVIEW

File: src/openzaak/components/besluiten/tests/test_besluit_create.py [hunk 1]
```diff
@@ -114,6 +114,7 @@ def test_us162_voeg_besluit_toe_aan_zaak(self):
                 besluit.besluitinformatieobject_set.get().informatieobject, io.canonical
             )
 
+    @override_settings(ALLOWED_HOSTS=["openzaak.nl", "testserver"])
```
Comment (sergei-maertens) [inline src/openzaak/components/besluiten/tests/test_besluit_create.py]: not sure what exactly makes these changes required?

Comment (stevenbal) [reply src/openzaak/components/besluiten/tests/test_besluit_create.py]: for some reason I was getting 400's due to disallowed host when testing locally, I'm not sure why that was happening though

Comment (sergei-maertens) [reply src/openzaak/components/besluiten/tests/test_besluit_create.py]: Can you try with a local `.env` file?

```bash
(open-zaak) ➜  open-zaak git:(master) cat .env
DEBUG=1
ALLOWED_HOSTS=localhost,127.0.0.1,testserver,openzaak.nl,example.com
DB_NAME=utrecht
```

Comment (stevenbal) [reply src/openzaak/components/besluiten/tests/test_besluit_create.py]: thanks, that fixes it

**Pull request merged**